packages/scaffold-config/src/dependencies.ts

```diff
@@ -76,7 +76,7 @@ export const WIZARD_DEPENDENCY_VITE = {
   package: 'vite',
   installer: 'vite',
   description: 'Vite is dev server that serves your source files over native ES modules',
-  minVersion: '^=2.0.0 || ^=3.0.0',
+  minVersion: '^=2.0.0 || ^=3.0.0 || ^=4.0.0',
```

Comment:
Not specific to this PR, just curious to get some thoughts - as we add more major versions to the support baseline do we want to continue this pattern or would something like this be easier to manage?
`>=2.0.0 <5.0.0`

Reply:
I guess that's easier, unless we encounter the rare situation where a single major isn't supported, but newer and older versions are (like with Node).

Reply:
Good point, there isn't an "exclude" operator so we'd end up having to stitch ranges together. In any event, not an issue at the moment just brainstorming a bit.